app/views/venues/show.html.erb

```diff
@@ -4,7 +4,7 @@
     <p><%= @venue.description %></p>
     <p><strong>Address:</strong> <%= @venue.address %></p>
     <p><strong>Capacity:</strong> <%= @venue.capacity %> people</p>
-    <p><strong>Bustability:</strong> <%= @venue.bustability %> % chance of being busted</p>
+    <p><strong>Bustability:</strong> <%= render "bustability", filled_hands: @filled_hands, empty_hands: @empty_hands %>
```

Comment:
where is @empty_hands defined ? 
does it appear already in the view like u want , if so lmk and ill merge